tests/core/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+cmake_policy(SET CMP0057 NEW)
```

Comment:
You ***must*** add a comment when playing with CMake policies. *Noone* knows that policy.